ledger/internal/jet/store.go

```diff
@@ -0,0 +1,156 @@
+/*
+ *    Copyright 2019 Insolar Technologies
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+package jet
+
+import (
+	"context"
+	"sync"
+
+	"github.com/insolar/insolar/core"
+)
+
+var (
```

Comment:
Maybe we should not be so strict about it.

Reply:
I found compile (IDE) time check id useful during development of implementation. Probably it could stay untouched until it would be a problem.